patches/vtk5.10.1VS2015.patch

```diff
@@ -0,0 +1,2654 @@
+diff --git a/Charts/vtkControlPointsItem.cxx b/Charts/vtkControlPointsItem.cxx
+index 7699abf..399f40d 100644
+--- a/Charts/vtkControlPointsItem.cxx
++++ b/Charts/vtkControlPointsItem.cxx
+@@ -32,6 +32,7 @@
+ 
+ #include <cassert>
+ #include <limits>
++#include <algorithm>
+ 
+ //-----------------------------------------------------------------------------
+ vtkControlPointsItem::vtkControlPointsItem()
+diff --git a/Charts/vtkPiecewisePointHandleItem.cxx b/Charts/vtkPiecewisePointHandleItem.cxx
+index 7dfa95c..63287cf 100644
+--- a/Charts/vtkPiecewisePointHandleItem.cxx
++++ b/Charts/vtkPiecewisePointHandleItem.cxx
+@@ -29,6 +29,8 @@
+ #include "vtkTransform2D.h"
+ #include "vtkNew.h"
+ 
++#include <algorithm>
++
+ enum enumPointHandleType
+   {
+   enSharpNess=0,
+diff --git a/Common/vtkOStreamWrapper.cxx b/Common/vtkOStreamWrapper.cxx
+index ff91c95..0df956e 100644
+--- a/Common/vtkOStreamWrapper.cxx
++++ b/Common/vtkOStreamWrapper.cxx
+@@ -57,7 +57,10 @@ VTKOSTREAM_OPERATOR(vtkObjectBase&);
+ VTKOSTREAM_OPERATOR(const vtkLargeInteger&);
+ VTKOSTREAM_OPERATOR(const vtkSmartPointerBase&);
+ VTKOSTREAM_OPERATOR(const vtkStdString&);
+-VTKOSTREAM_OPERATOR(ostream&);
++vtkOStreamWrapper& vtkOStreamWrapper::operator << (ostream& a) {
++  this->ostr << (void *)&a;
++  return *this;
++}
+ VTKOSTREAM_OPERATOR(const char*);
+ VTKOSTREAM_OPERATOR(void*);
+ VTKOSTREAM_OPERATOR(char);
+diff --git a/Hybrid/vtk3DSImporter.cxx b/Hybrid/vtk3DSImporter.cxx
+index bfa0996..efcf3af 100644
+--- a/Hybrid/vtk3DSImporter.cxx
++++ b/Hybrid/vtk3DSImporter.cxx
+@@ -39,11 +39,11 @@ vtkStandardNewMacro(vtk3DSImporter);
+ #pragma GCC diagnostic ignored "-Wstrict-aliasing"
+ #endif
+ 
+-static vtk3DSColour Black = {0.0, 0.0, 0.0};
++static vtk3DSColour Black = {0.0f, 0.0f, 0.0f};
+ static char   obj_name[80] = "";
+-static vtk3DSColour fog_colour = {0.0, 0.0, 0.0};
+-static vtk3DSColour col        = {0.0, 0.0, 0.0};
+-static vtk3DSColour global_amb = {0.1, 0.1, 0.1};
++static vtk3DSColour fog_colour = {0.0f, 0.0f, 0.0f};
++static vtk3DSColour col        = {0.0f, 0.0f, 0.0f};
++static vtk3DSColour global_amb = {0.1f, 0.1f, 0.1f};
+ static vtk3DSVector pos        = {0.0, 0.0, 0.0};
+ static vtk3DSVector target     = {0.0, 0.0, 0.0};
+ static float  hotspot = -1;
+diff --git a/Hybrid/vtkVectorText.cxx b/Hybrid/vtkVectorText.cxx
```

Comment:
i took the original ITK 4.11.1 file directly on their github because there are a lot of changes - mostly double->float.